### PR TITLE
Refactoring by pasha

### PR DESCRIPTION
**Pull request by SoloPashaS**
deleted all ICollection<> from classes

### CODE REVIEW

File: DBlab2/DBlab2/Group.cs [hunk 1]
```diff
@@ -3,12 +3,10 @@
 namespace DBlab2 {
     public class Group {
         public int Id { get; set; }
-        public int Number { get; set; }
+        public int? Number { get; set; }
         public int SpecialityId { get; set; }
         public int CathedraId { get; set; }
 
-        public virtual Speciality Speciality { get; set; }
```
Comment (ArthurMamedov) [inline DBlab2/DBlab2/Group.cs:10]: Why did you delete this? Return this, please...

**Review by ArthurMamedov — CHANGES_REQUESTED** [review]

File: DBlab2/DBlab2/Group.cs [hunk 2]
```diff
@@ -8,5 +8,6 @@ public class Group {
         public int CathedraId { get; set; }
 
         public virtual Cathedra Cathedra { get; set; }
+        public virtual Speciality Speciality { get; set; }
```
Comment (ArthurMamedov) [inline DBlab2/DBlab2/Group.cs:11]: Ok, well... Good Job!

**Review by ArthurMamedov — APPROVED** [review]

File: DBlab2/DBlab2/Group.cs (hunk 2 above)
Comment (SoloPashaS) [reply DBlab2/DBlab2/Group.cs:11]: thank you!
